visual-js/visual-storybook/src/api.ts

```diff
@@ -143,3 +145,32 @@ export const postVisit = async (page: Page, context: TestContext) => {
  * `postVisit` exported from this package instead.
  */
 export const postRender = postVisit;
+
+let hasExposed = false;
```

Comment:
Could you add a comment about the intent and lifecycle of this?

Reply:
Added